src/pages/besquare/components/helper/_card.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import {
+    Bullet,
+    CardWrapper,
+    DropdownWrapper,
+    IconWrapper,
+    ListContainer,
+} from '../../static/style/_card'
+import { TextWrapper } from '../../static/style/_common'
+import { Minimize, Maximize } from '../../static/images/_what-lies-ahead'
+
+const Card = ({ card_content, custom_icon, has_list, style, title_component }) => {
+    const [is_list_open, setIsListOpen] = useState(false)
+
+    const getCurrentDropdownComponent = () => {
+        if (is_list_open) {
+            return (
+                <React.Fragment>
+                    <DropdownWrapper
```

Comment:
[P1]
You can store components in a variable and reuse it at the bottom. Eg;
`const components = <DropdownWrapper .../>`